content/spin/http-outbound.md

```diff
@@ -142,7 +142,7 @@ By default, Spin components are not allowed to make outgoing HTTP requests. This
 
 ```toml
 [component]
-allowed_http_hosts = [ "dog-facts.example.com", "api.example.com:8080" ]
+allowed_http_hosts = [ "random-data-api.fermyon.app", "api.example.com:8080" ]
```

Comment:
This looks like it intentionally did not use the API domain - it was using `example.com`, I guess to emphasise that this was an example not a usable fragment.